### ISSUE TITLE
High level image classification module

### ISSUE THREAD

**Issue by YanYas** [ThinkingForms/VLML#2]
Need to finish the high-level module you have for training image classification.

**Comment by YanYas** [comment]
One will be provided in the examples, but I want to avoid having large model binaries in the basic package. A downloader should do the trick for the most part.

**Comment by sebllll** [comment]
> One will be provided in the examples, but I want to avoid having large model binaries in the basic package. A downloader should do the trick for the most part.

what about git lfs? i think there's a 1GB quota for free accounts.

**Comment by YanYas** [comment]
Indeed. There is a limit on the size of a Nuget, maybe 250MB, that I want to stay under too, but it may be alright. There are a couple different models that could be particularly helpful. Resnet34 which is both relatively small (~70MB) and very accurate, and VGG/16/19 which is around 500MB, is not as accurate as Resnet, but has other important uses, such as Style Transfer for unpicking the semantics of an image, I'll have a look at other forms of storage first, but  will try to move ahead on this.

**Comment by sebllll** [comment]
sorry, i misread your comment and was thinking that you want to have the repo small, but you're talking about the nuget which is a separate issue.

in this case, i don't think any large files should be included in the nuget. 

i'd prefer smth. like [this ](https://mastodon.xyz/@vvvv/103494838093976217) and download files in runtime.

**Comment by YanYas** [comment]
Good example. Yeah I have something very similar in the [VLMLDemos](https://github.com/ThinkingForms/VLML_Demos_V1), but I'll make it a dependency for the main package so I can update it separately. Thanks for mentioning!
